service/standalone_apis_test.go

```diff
@@ -177,6 +174,52 @@ func TestRevokeHandler(t *testing.T) {
 		require.Equal(t, http.StatusOK, w.Result().StatusCode)
 	})
 
+	t.Run("performs correct delete query only on resource type and id", func(t *testing.T) {
+		bindingsFromCrud := []types.Binding{
+			{
+				BindingID: "bindingToDelete",
+				Subjects:  []string{"piero"},
+				Resource: &types.Resource{
+					ResourceType: "project",
+					ResourceID:   "mike",
+				},
+			},
+		}
+		gock.DisableNetworking()
+		newGockScope(t, "http://crud-service", http.MethodGet, "/bindings/").
+			AddMatcher(func(req *http.Request, greq *gock.Request) (bool, error) {
+				expected := `{"$and":[{"resource.resourceId":{"$in":["mike"]},"resource.resourceType":"myResource"},{"$or":[]}]}`
```

Comment:
why the `{"$or":[]}` ? can we remove this query token?

Reply:
Sure. It is still there because i haven't touched the query generation logic

Reply:
Empty or removed